TWLight/users/oauth.py

```diff
@@ -272,6 +272,12 @@ def authenticate(self, request=None, access_token=None, handshaker=None):
 
         request.session["user_created"] = created
 
+        # Checking if more than one session exists
+        sessions = Session.objects.filter(account_id=user.pk)
+        if sessions.count() >= 1:
+            logger.info("Deleting all previous sessions.")
+            sessions.delete()
```

Comment:
I have a question about deleting all sessions by account_id-  will this affect the user's current session?

Reply:
It does not, otherwise, you would enter an infinite log-in loop. I also thought that at first, so I excluded the latest session from my delete query, but two sessions would remain in the /admin. I think the session is created a bit after authentication.

Reply:
Ok, sounds good, that's what I was thinking too but wanted to verify!